pages/common/gdu.md

```diff
@@ -0,0 +1,36 @@
+# gdu
+
+> Disk usage analyzer with console interface.
+> More information: <https://github.com/dundee/gdu>.
+
+- Show disk usage of the current directory:
+
+`gdu`
+
+- Show disk usage of given directory:
+
+`gdu {{path/to/directory}}`
+
+- Show disk usage of all mounted disks:
+
+`gdu --show-disks`
+
+- Show disk usage but ignore some paths:
+
+`gdu --ignore-dirs {{/sys,/proc}} /`
+
+- Ignore paths by regular pattern:
+
+`gdu --ignore-dirs-pattern {{'.*[abc]+'}}`
+
+- Ignore hidden directories:
+
+`gdu --no-hidden`
+
+- Only print result, do not enter interactive mode:
```

Comment:
So all the other examples are interactive? In which way is it interactive?

Reply:
I guess it's like `fdisk` interactive mode.

Reply:
Then we should change them all to `Interactively show the disk usage ...`.